src/main/java/org/openrewrite/kotlin/internal/KotlinTreeParserVisitor.java

```diff
@@ -1698,8 +1703,8 @@ public J visitAnnotation(KtAnnotation annotation, ExecutionContext data) {
 
         return mapType(new J.Annotation(randomId(),
                 Space.EMPTY,
-                Markers.EMPTY.addIfAbsent(new AnnotationUseSite(randomId(), suffix(annotation.getUseSiteTarget()), isImplicitBracket)),
-                (NameTree) annotation.getUseSiteTarget().accept(this, data),
+                Markers.EMPTY,
+                new K.UseSite(randomId(), Space.EMPTY, Markers.EMPTY, padRight(target, suffix(annotation.getUseSiteTarget())), isImplicitBracket),
```

Comment:
Do I understand it correctly that now every annotation will have a `K.UseSite` and that this will wrap a `J.Empty` when there wasn't any use-site target? That would be wrong, I think.

Reply:
Not every annotation will have `K.UseSite`, but just use-site annotations which include below 4 cases
```
@[foo, bar]     // This will have a `J.Empty` as the target
@field: foo
@field: [ foo ]
@field: [ foo bar]
```

Reply:
the regular annotation is still a `J.Annotation` with no change and no `K.UseSite` there, `K.UseSite` is for use-site annotations only